### PR TITLE
Juniper 2020 01 08

### PR DESCRIPTION
**Pull request by 0marg**
Hello, this is Juniper SIRT.
Please merge Juniper 2020-01-08 branch to the CVEProject.

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1600.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1600.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1601.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1601.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1602.json [hunk 3]
```diff
@@ -0,0 +1,235 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1602.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1603.json [hunk 4]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1603.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1604.json [hunk 5]
```diff
@@ -0,0 +1,180 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1604.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1605.json [hunk 6]
```diff
@@ -0,0 +1,227 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1605.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1606.json [hunk 7]
```diff
@@ -0,0 +1,216 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1606.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1607.json [hunk 8]
```diff
@@ -0,0 +1,216 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1607.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1608.json [hunk 9]
```diff
@@ -0,0 +1,200 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1608.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1609.json [hunk 10]
```diff
@@ -0,0 +1,227 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1609.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1611.json [hunk 11]
```diff
@@ -0,0 +1,106 @@
+{
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1611.json]: Invalid JSON file

File: 2020/1xxx/CVE-2020-1600.json (hunk 1 above)
Comment (mpaujnpr01) [reply 2020/1xxx/CVE-2020-1600.json]: According to the review of https://jsonformatter.org/ this JSON format is valid.  Please let us know what is wrong with not having a CRLF.  We're curious if this automation validation error is at MITREs end.  If not, why are we seeing this now, when we've not seen this problem before?  Has there been a standards change we're unaware of?

Comment (EvansJonathan) [reply 2020/1xxx/CVE-2020-1600.json]: The problem isn't the CRLF.  You have an invalid STATE value.  The error check doesn't handle these types of issue well.

**Comment by mpaujnpr01** [comment]
Ah, you’re reviewing content as well as format; the error report looked like it was the end line…  ok, let me check that.
Thank you Jonathan.


Sincerely,
Matt Paulsen
mpaulsen@juniper.net<mailto:mpaulsen@juniper.net>
o +1-408-933-6323<tel:14089336323>
Sr. incident Manager, Security Incident Response
Juniper Security Incident Response Team
PGP Key ID 1AED3D24
Fingerprint D4FE E0A3 A482 F7C6 C687 466E F322 3495 1AED 3D24
Verify: https://keys.openpgp.org/verify/Tnj0as4TShDPSUbpPVOdRp2iCyGwMmHbca928ikO6c6<https://urldefense.proofpoint.com/v2/url?u=https-3A__keys.openpgp.org_verify_Tnj0as4TShDPSUbpPVOdRp2iCyGwMmHbca928ikO6c6&d=DwMBaQ&c=HAkYuh63rsuhr6Scbfh0UjBXeMK-ndb3voDTXcWzoCI&r=eoE6DtCerCdL-MhOC3m_A5UTC4sSV_5QscncrhDl2Pc&m=xc2C-Tztnai_mMDKuCVK_XzmkDx5kr8c6222yzCJjNM&s=RuLac_vW4Gpf1L1lm-WPGvM4APQZwPdrLkfX9-gsVB8&e=>

Please report any potential or real instances of security vulnerabilities with any Juniper Networks product to the Juniper Networks Security Incident Response Team at sirt@juniper.net<mailto:sirt@juniper.net>.  To use PGP please download, install the following apps and services and provide a public key to us.  We suggest using GPGTools/GNUPG with Thunderbird and Enigmail.  https://gpgtools.org   https://www.thunderbird.net/en-US/   https://www.enigmail.net/index.php/en/home  https://gnupg.org/  For non-product security vulnerability incidents, you can receive immediate assistance by contacting JTAC 24 hours a day by calling 888-314-JTAC FREE (North America) or +1-408-745-9500.

**Pull request merged**